recipes/r-metaskat/meta.yaml

```diff
@@ -36,8 +36,10 @@ requirements:
   run:
     - r-base
     - r-skat
-    - libstdcxx-ng                     # [not win]
-    - libgcc-ng                        # [not win]
+    - clang_osx-64                     # [osx]
+    - clangxx_osx-64                   # [osx]
+    - gcc_linux-64                     # [linux64]
+    - gxx_linux-64                     # [linux64]
```

Comment:
Are these dependencies really necessary? I'd think they're already pulled in by the compiler bits below.